modules/security-context-constraints-about.adoc

```diff
@@ -109,12 +110,12 @@ The `privileged` SCC allows:
 
 [NOTE]
 ====
-Setting `privileged: true` in the pod specification does not select the `privileged` SCC. Setting `privileged: true` in the pod specification matches on the `allowPrivilegedContainer` field of an SCC.
+Setting `privileged: true` in the pod specification does not necessarily select the `privileged` SCC. The SCC which has `allowPrivilegedContainer: true` and has the highest prioritization will be chosen if the user has the permission to use it.
 ====
 endif::[]
 
 |`restricted`
-|Denies access to all host features and requires pods to be run with a UID, and SELinux context that are allocated to the namespace.  This is the most restrictive SCC and it is used by default for authenticated users.
+|Denies access to all host features and requires pods to be run with a UID, and SELinux context that are allocated to the namespace.  This is the most restrictive default SCC and it is used by default for authenticated users.
```

Comment:
I suspect that "default" isn't the right word. We can't have multiple defaults for the same thing, right?

Perhaps you mean something closer to standard, prebuilt, preset?

Reply:
@dbaker-rh could you please pitch in and clarify @ctauchen question.

Reply:
Yes.  This phrase, however inaccurate, is used in our own docs already: https://docs.openshift.com/container-platform/4.9/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth

"The cluster contains several **default security context constraints** (SCCs) as described in the table below."

The intent of my change was to separate "most restrictive possible" (as is implied) from "most restrictive if you've not made any changes" (which is true).

To take a bash at rephrasing this:

>> ... This is the most restrictive SCC provided by a new installation and will be used by default for authenticated users.


If that gets too convoluted then leaving in it's mildly inaccurate current state might be better than explaining it to death.

Reply:
Ok, we'll go with the usage from the docs. I like that phrasing: 

This is the most restrictive SCC provided by a new installation and will be used by default for authenticated users.

Reply:
@ctauchen Done!